api/v1alpha1/ingressnodefirewall_types.go

```diff
@@ -77,63 +85,61 @@ type IngressNodeFirewallRuleProtocolType string
 
 const (
 	// ProtocolTypeICMP refers to the ICMP protocol
-	ProtocolTypeICMP IngressNodeFirewallRuleProtocolType = "icmp"
+	ProtocolTypeICMP IngressNodeFirewallRuleProtocolType = "ICMP"
 
 	// ProtocolTypeICMP6 refers to the ICMPv6 protocol
-	ProtocolTypeICMP6 IngressNodeFirewallRuleProtocolType = "icmp6"
+	ProtocolTypeICMP6 IngressNodeFirewallRuleProtocolType = "ICMPv6"
 
 	// ProtocolTypeTCP refers to the TCP protocol, for either IPv4 or IPv6
-	ProtocolTypeTCP IngressNodeFirewallRuleProtocolType = "tcp"
+	ProtocolTypeTCP IngressNodeFirewallRuleProtocolType = "TCP"
 
 	// ProtocolTypeUDP refers to the UDP protocol, for either IPv4 or IPv6
-	ProtocolTypeUDP IngressNodeFirewallRuleProtocolType = "udp"
+	ProtocolTypeUDP IngressNodeFirewallRuleProtocolType = "UDP"
 
 	// ProtocolTypeSCTP refers to the SCTP protocol, for either IPv4 or IPv6
-	ProtocolTypeSCTP IngressNodeFirewallRuleProtocolType = "sctp"
+	ProtocolTypeSCTP IngressNodeFirewallRuleProtocolType = "SCTP"
 )
 
 // IngressNodeFirewallActionType indicates whether an IngressNodeFirewallRule allows or denies traffic
-// +kubebuilder:validation:Pattern=`^allow|deny$`
+// +kubebuilder:validation:Enum="Allow";"Deny"
 type IngressNodeFirewallActionType string
 
 const (
-	IngressNodeFirewallAllow IngressNodeFirewallActionType = "allow"
-	IngressNodeFirewallDeny  IngressNodeFirewallActionType = "deny"
+	IngressNodeFirewallAllow IngressNodeFirewallActionType = "Allow"
+	IngressNodeFirewallDeny  IngressNodeFirewallActionType = "Deny"
 )
 
 // IngressNodeFirewallRules define ingress node firewall rule
 type IngressNodeFirewallRules struct {
 	// SourceCIDRS is A list of CIDRs from which we apply node firewall rule
```

Comment:
`SourceCIDRs defines the origin of packets that FirewallProtocolRules will be applied to.`